testing/tests/DevExpress.ui.events/click.tests.js

```diff
@@ -58,10 +58,10 @@ QUnit.test("event triggers", function(assert) {
 });
 
 QUnit.test("event args", function(assert) {
-    var fields = ["altKey", "button", "cancelable", "clientX", "clientY",
+    var fields = ["altKey", "cancelable", "clientX", "clientY",
```

Comment:
We remove 'button' and 'relatedTarget' properties to check only common properties for touch and mouse events.